<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>itemMenu</class>
 <widget class="QWidget" name="itemMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QPushButton" name="rotateAClockwiseBtn">
       <property name="text">
        <string>Rotate AntiClockwise</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="coordinateInput"/>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="xNegBtn">
       <property name="text">
        <string>-x</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="rotateClockwise">
       <property name="text">
        <string>Rotate Clockwise</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="zPosBtn">
       <property name="text">
        <string>+z</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="rotateUpBtn">
       <property name="text">
        <string>RotateUp</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="xPosBtn">
       <property name="text">
        <string>+x</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="angleInput"/>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="zNegBtn">
       <property name="text">
        <string>-z</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="rotateDownBtn">
       <property name="text">
        <string>RotateDown</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
